components/workshops.tsx

```diff
@@ -1,9 +1,80 @@
-const Workshops = () => (
-  <div className="bg-red-400">
-    <div className="flex flex-col sm:flex-row p-6 sm:p-12">
-      <h1 className="text-5xl sm:text-9xl font-bold">2. Workshops</h1>
+import WorkshopCard from "./card"
+import { useState, useEffect } from "react";
+import IEvent from "../utils/IEvent";
+
+const Workshops = () => {
+  const [events, setEvents] = useState<IEvent[]>([]);
+
+  const fetchEvents = async() => {
+    const response = await fetch('/api/workshop', {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json'
+      },
+    });
+    console.log(response)
+
+    if (!response.ok) {
+      throw new Error(`Error: ${response.status}`);
+    }
+
+    const fetchedEvents = await response.json();
+    setEvents(fetchedEvents.slice(0,3));
+  }
+
+  useEffect(() => {
+    fetchEvents();
+  }, []);
+
+  // console.log(events)
+
+  return(
+    <div className="bg-[url('/img/news_paper_bg.jpg')] bg-cover font-alegre">
+      <div className="min-h-screen">
+        <div className="mt-3">
+          <div className="grid grid-cols-5">
+            <div className="p-5">
+              <div className="p-2 border-4 border-slate-800">
+                <h1 className="font-bold">Today's Weather:</h1>
```

Comment:
We should store a few different values for the weather in a file and then randomly choose one on page refresh :P kinda like [the `footerPonderings` on the events site](https://github.com/purduehackers/events/blob/main/components/upcoming-event.tsx#L20).